config/services/dataplane_v1beta1_openstackdataplaneservice_nova.yaml

```diff
@@ -3,12 +3,15 @@ kind: OpenStackDataPlaneService
 metadata:
   name: nova
 spec:
-  secrets:
-    - nova-cell1-compute-config
-    # NOTE: this Secret needs to be created before deploying the data plane.
-    # It should contain an ssh key-pair in the secret fields: ssh-privatekey
-    # and ssh-publickey
-    - nova-migration-ssh-key
+  secrets: []
+  datasources:
```

Comment:
ah i lost james fix this shoudl be dataSources i belive

Reply:
yep
https://github.com/openstack-k8s-operators/dataplane-operator/blob/6c7c31068c9f936113aeec1b60cc278010ef932e/api/v1beta1/openstackdataplaneservice_types.go#L71C1-L71C57

im not sure that is why the job failed as extra filed should be droped by the api but ill fix this and lest it run over night and see if it passes in the morning

Reply:
ya looks like the job failure was unrealated

https://review.rdoproject.org/zuul/build/d88b1ded972f4b9abacc56f7e1a0aad2/log/controller/ci-framework-data/logs/openstack-k8s-operators-openstack-must-gather/namespaces/openstack/pods/repo-setup-edpm-deployment-openstack-edpm-ipam-vrt9w/logs/openstackansibleee.log

the repo setup task failed.

the repo setup openstack service looks broken to me

https://review.rdoproject.org/zuul/build/d88b1ded972f4b9abacc56f7e1a0aad2/log/controller/ci-framework-data/logs/openstack-k8s-operators-openstack-must-gather/namespaces/openstack/crs/openstackdataplaneservices.dataplane.openstack.org/repo-setup.yaml

it shoudl have a play defiend inline but does not

https://github.com/openstack-k8s-operators/install_yamls/blob/main/devsetup/edpm/services/dataplane_v1beta1_openstackdataplaneservice_reposetup.yaml#L14